lib/schedule/minischedule/piece.ex

```diff
@@ -89,8 +88,7 @@ defmodule Schedule.Minischedule.Piece do
         ) :: mid_route_swing()
   defp hydrate_mid_route_swing(mid_route_swing, trips_by_id, timepoint_names_by_id) do
     trip_id = mid_route_swing.trip
-    full_trip = trips_by_id[trip_id]
-    minischedule_trip = Trip.from_full_trip(full_trip, timepoint_names_by_id)
-    %{mid_route_swing | trip: minischedule_trip}
+    trip = trips_by_id[trip_id] |> Trip.set_pretty_names(timepoint_names_by_id)
```

Comment:
IIRC we try to avoid single-pipe pipelines and just write out the function call conventionally instead.

Reply:
_Although_, it may be preferable to use `Map.get` anyway.